manual/src/cmds/unified-options.etex

```diff
@@ -851,5 +851,27 @@ hyphen (-).}
 \item["-help" or "--help"]
 Display a short usage summary and exit.
 
+\item[(JST) "-extension" \var{language-extension}]
+Enable the specified \var{language-extension}.  Can be specified more than once,
+either with the same or a different language extension; is idempotent.
+
+\item[(JST) "-no-extension" \var{language-extension}]
+Disable the specified \var{language-extension}.  Can be specified more than once,
+either with the same or a different language extension; is idempotent.
+
+\item[(JST) "-only-erasable-extensions"]
+Restricts the "-extension" and "-no-extension" options to work only with
+so-called ``erasable'' extensions: ones that can be rewritten into attributes
+while still preserving the program's observable runtime behavior.  Turns off
```

Comment:
There's a lot going into "observable" here. The amount of allocations is an observable quantity, for a sufficiently astute observer. Maybe change "observable" to "input/output"?

Reply:
e64e0bde88